subsys/bluetooth/host/att.c

```diff
@@ -2622,6 +2623,7 @@ static void att_reset(struct bt_att *att)
 		net_buf_unref(buf);
 	}
 
+	conn = att->conn;
 	att->conn = NULL;
```

Comment:
How is `att->conn` reference counted? It looks like there's a missing `bt_conn_unref()` before the original `att->conn = NULL`. If `att->conn` wasn't ref-counted (created with `att->conn = bt_conn_ref(conn);` somewhere) that should probably be fixed.

Reply:
@jhedberg I think this issue should not block this PR in any case.